src/automatic-annotations/unirule/components/entry/Entry.tsx

```diff
@@ -0,0 +1,64 @@
+import { RouteChildrenProps } from 'react-router-dom';
+import { Loader, Card } from 'franklin-sites';
+import cn from 'classnames';
+
+import HTMLHead from '../../../../shared/components/HTMLHead';
+import SingleColumnLayout from '../../../../shared/components/layouts/SingleColumnLayout';
+import ErrorHandler from '../../../../shared/components/error-pages/ErrorHandler';
+
+import { MapToDropdown } from '../../../../shared/components/MapTo';
+
+import useDataApiWithStale from '../../../../shared/hooks/useDataApiWithStale';
+
+import apiUrls from '../../../../shared/config/apiUrls';
+
+import {
+  Namespace,
+  SearchableNamespaceLabels,
+} from '../../../../shared/types/namespaces';
+import { UniRuleAPIModel } from '../../adapters/uniRuleConverter';
+
+import helper from '../../../../shared/styles/helper.module.scss';
+import entryPageStyles from '../../../shared/styles/entry-page.module.scss';
+
+const UniRuleEntry = (props: RouteChildrenProps<{ accession: string }>) => {
+  const accession = props.match?.params.accession;
+
+  const { data, loading, error, status, progress, isStale } =
+    useDataApiWithStale<UniRuleAPIModel>(
+      apiUrls.entry(accession, Namespace.unirule)
+    );
+
+  if (error || !accession || (!loading && !data)) {
+    return <ErrorHandler status={status} />;
+  }
+
+  if (!data) {
+    return <Loader progress={progress} />;
+  }
+
+  return (
+    <SingleColumnLayout>
+      <HTMLHead
+        title={[data.uniRuleId, SearchableNamespaceLabels[Namespace.unirule]]}
+      />
+      {/* Here we don't want to use the full label atm */}
+      <h1 className="big">
+        {SearchableNamespaceLabels[Namespace.unirule]} - {data.uniRuleId}
+      </h1>
+      <Card className={cn(entryPageStyles.card, { [helper.stale]: isStale })}>
+        <div className="button-group">
+          <MapToDropdown
+            statistics={{
+              reviewedProteinCount: 0,
```

Comment:
This is blocked by https://www.ebi.ac.uk/panda/jira/browse/TRM-26560 at the moment, so will need to be updated whenever it's done

Reply:
So this is just a placeholder value for now

Reply:
Is this likely to be updated soon? If not, could use a TODO

Reply:
Not sure, the ticket is marked as resolved but we're still pending deployement by backend. So I'm assume that yes, it will be soon. Added TODO comments anyway